src/test/java/api/loans/scenarios/ChangeDueDateTests.java

```diff
@@ -1,31 +1,25 @@
 package api.loans.scenarios;
 
-import static api.support.http.InterfaceUrls.loansUrl;
 import static api.support.matchers.PatronNoticeMatcher.hasEmailNoticeProperties;
 import static api.support.matchers.TextDateTimeMatcher.isEquivalentTo;
-import static java.net.HttpURLConnection.HTTP_NO_CONTENT;
 import static org.folio.circulation.support.JsonPropertyWriter.write;
 import static org.hamcrest.CoreMatchers.hasItems;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.junit.MatcherAssert.assertThat;
 
-import java.net.MalformedURLException;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
 
+import api.support.builders.DeclareItemLostRequestBuilder;
```

Comment:
Please can these imports be re-ordered to match the provisionally agreed ordering.

If you are using the Eclipse formatter plugin, please refer to the config file available at https://issues.folio.org/browse/FOLIO-1667 for how to re-order these statements without the plugin.

@roman-barannyk and @mrdpopov have also recently encountered this issue, maybe they can help advise how they resolved it.

Reply:
Yes, @roman-barannyk helped me with this. I think I fixed it.